relentless/potential/potential.py

```diff
@@ -36,12 +36,7 @@ class CoefficientMatrix(PairMatrix):
     --------
     Create a coefficient matrix with defined types and params::
 
```

Comment:
Please review this doc and remove any refs to default values. (There is `default` in the Parameters list above, and in the examples below)